frontend/src/metabase-lib/v1/expressions/suggest.ts

```diff
@@ -195,6 +195,12 @@ export function suggest({
         .filter(
           clause => clause && database?.hasFeature(clause.requiresFeature),
         )
+        .filter(function disableOffsetInFilterExpressions(clause) {
+          const isOffset = clause.name === "offset";
```

Comment:
nit: I'd separate this function `disableOffsetInFilterExpressions` and call it in the prev filter through `&&`

Reply:
I'll leave it like that since it's a bit less convoluted this way.
And ideally we'll be removing that code block in v51.
There's no performance issue here :)